pkgs/development/libraries/librime/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchFromGitHub, cmake, boost, glog, leveldb, marisa, opencc,
-  yaml-cpp, gtest, capnproto, pkg-config, plugins ? [ ] }:
+  yaml-cpp, gtest, capnproto, pkg-config, librime-lua, plugins ? [ librime-lua ] }:
```

Comment:
The first `librime-lua` is not necessary. Adding it to `plugins` is sufficient.

Reply:
Removing the first `librime-lua` causes an error:

```console
error: undefined variable 'librime-lua'
```

Reply:
Oh, I see what you're doing here. Your approach is correct, please keep both of them in place.